docs/styles/style.css

```diff
@@ -29,6 +33,39 @@ p {
   font-size: calc(var(--typo-scale) * 1rem);
 }
 
+ul, ol {
+  padding-left: 1.5rem;
+}
+
+@media screen and (min-width: 481px) {
```

Comment:
If you look in your browser in the mobile/responsive view you notice that it doesn't behave the same way like you minimize the window. 


To have the same you need to have an additional meta-tag 

![image](https://user-images.githubusercontent.com/992878/95451541-3b3c7a00-0968-11eb-9deb-96c65e016f58.png)

https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag